src/plugin/ipc/file/fileconnection.cpp

```diff
@@ -251,12 +261,24 @@ void FileConnection::refill(bool isRestart)
                 "   Change this function and re-compile, if you want "
                 "different behavior.");
       } else {
-        const char *errMsg =
+        char *errMsg =
```

Comment:
I'd say that the comparison allows us to save time on copying data from the checkpointed version to the existing version when the contents match.